src/lib/index.js

```diff
@@ -1,2 +1,6 @@
+export { documentApi } from '@api/documents';
```

Comment:
It would be great if you export all APIs... basically:
* all APIs
* history
* App
* store
* the components you need for now

Can you please delete the file `src/lib/api/index.js` and fix any import if it breaks?